nodes.py

```
@@ -1240,6 +1506,83 @@ def post(self, shared, prep_res, exec_res):
+        csv_result = prep_res["csv_result"]
+        api_result = prep_res["api_result"]
+        discrepancies = []
+        reconciled = {}
+
+        if isinstance(csv_result, dict) and isinstance(api_result, dict):
+            keys = set(csv_result.keys()) | set(api_result.keys())
+            for key in keys:
+                csv_val = csv_result.get(key)
+                api_val = api_result.get(key)
+                diff_pct, severity = self._compare_scalars(csv_val, api_val)
+                preferred, source = data_source_manager.reconcile_conflicts(csv_val, api_val, key)
```

Ruff (0.14.10)

Unpacked variable `source` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)